test/moduleId.html

```diff
@@ -5,6 +5,9 @@
 		<title>QUnit ModuleId Test Suite</title>
 		<link rel="stylesheet" href="../dist/qunit.css">
 		<script src="../dist/qunit.js"></script>
+		<script>
+			QUnit.warning.off();
+		</script>
```

Comment:
Following Warnings: 
```
DEPRECATED: duration property will be removed in 2.0.0, use runtime instead.
DEPRECATED: Use multiple tests instead of resetting inside a test. Use testStart or testDone for custom cleanup. This method will throw an error in 2.0, and will be removed in 2.1.```
```

Doesn't make sense throw this warnings?